spec/support/feature_helpers.rb

```diff
@@ -0,0 +1,10 @@
+module FeatureHelpers
+
+  def login_user(username, password)
```

Comment:
You could probably make this a little easier to use by taking an actual `User` record as the parameter.  Then you're able to both set a default for that argument as well as peel out the `username` and `password` off the given user:
```ruby
def login_user(user = nil)
  user ||= User.create(username: 'test', password: 'test', email: 'test@test.com')

  visit login_path

  fill_in 'Username', with: username.to_s
  fill_in 'Password', with: password.to_s

  click_button 'Login'
end
```

This would let you -- in cases where you don't necessarily care about what user is logging in -- to login like:
```
before do
  login_user
end
```